getsolus/packages
# g4music: Initial Inclusion

**Summary**

- Initial Inclusion of g4music
- Resolves #934 

**Test Plan**

- Load music library
- Play a couple of music
- Close window to check background play

**Checklist**
